### DISCUSSION TITLE
Fragment testing and Robolectric

### DISCUSSION THREAD

**Discussion by JavierSegoviaCordoba** [sczerwinski/android-hilt#12] [Q&A] [answered]
Is it compatible with Robolectric? I think it isn't because it needs to use `debugImplementation`. Is there a workaround?

**Comment by sczerwinski** [comment, marked as answer]
No, it is not. `ActivityScenario` and `FragmentScenario` are designed for connected Android tests.

For using Hilt with Robolectric, there is different setup: https://dagger.dev/hilt/robolectric-testing
